cogs/autoVoiceChannels.py

```diff
@@ -1,5 +1,5 @@
 
-import nextcord
+import nextcord, re
```

Comment:
What is the point of importing re?

Reply:
line 138 number = int(re.search(r'\d+', channelName).group())